examples/models/models_first_person_maze.ml

```diff
@@ -0,0 +1,143 @@
+type gui_state = {
+  camera : Raylib.Camera3D.t;
+  cubicmap : Raylib.Texture2D.t;
+  model : Raylib.Model.t;
+  texture : Raylib.Texture2D.t;
+  map_pixels : Raylib.Color.t Raylib.CArray.t;
+  map_position : Raylib.Vector3.t;
+}
+
+let gui_init () =
+  let open Raylib in
+  init_window 800 600 "Reactive GUI";
```

Comment:
Please use the name of the example for the window.
`"raylib [models] example - first person maze"`

Reply:
Thank you! Copy/pasted from the neighbor test project with RayGUI and React. Fixed.